.gitignore

```diff
@@ -1,4 +1,5 @@
 .stylelintrc.json
 .hintrc
 .eslintrc.json
```

Comment:
Kindly remove your linting config files from the `.gitignore` file,

Reply:
@abruzy these are local linters. I have an option to ignore them.

Reply:
@the-catalystmc you don't need to ignore them. you only need to ignore your `node_modules` folder

Reply:
I have been ignoring these since the start of the program as was advised. They are local, I have been through this with another reviewer already man. I am tired.

Reply:
Hi @the-catalystmc, really sorry, I know it can be very tiring and frustrating at the same time but we are only trying to make you a better developer by enforcing you to follow best practices. Just try and fix this and other changes and you'll be all good, you just need those and get your project approved. if you can fix those and resubmit, I'll review it again and get your project right away if everything else looks good.

Reply:
@ abruzy please explain why the below are important. My project works fine the way it is.
"scripts": {
"dev": "webpack --mode development",
"build": "webpack --mode production",
"start": "webpack --config webpack.config.js --watch"
}

Reply:
Hi @the-catalystmc  again,

Those are required to help you differentiate between a development build and production builds, for example, you want to deploy your app to production using one of the online hosting platforms like netlify or Heroku, they'll look for your build scripts and run, and you know your production build is always different from your development builds, so the reason why you did to add mode to your webpack scripts, I hope this explains it to you.